.changeset/happy-items-turn.md

```diff
@@ -0,0 +1,7 @@
+---
+'@makeswift/runtime': minor
+---
+
+BREAKING: The `MakeswiftClient.prefetch()` and `MakeswiftClient.getTypographies` methods have been removed. These were internal APIs so there shouldn't be any changes required to upgrade.
```

Comment:
Should be `fetchTypographies`?

Reply:
Yes! Nice catch.

As to why we're only removing `fetchTypographies` it's because that one's only used for introspection so it's being moved to the Makeswift API client. All other fetch methods are still needed by the host API client to fetch data for controls like swatches, files, etc.